### PR TITLE
camera editor multi editing

### PR DESCRIPTION
**Pull request by martint-unity**
---
### Purpose of this PR
This is a fix for this FB Case
https://fogbugz.unity3d.com/f/cases/1230080

Where multi Editing of Camera properties didn't work. 
Please re-open old projects and make sure that no serialized data is lost.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs [hunk 1]
```diff
@@ -113,15 +113,12 @@ public static string GetName(this CameraRenderType type)
     [ImageEffectAllowedInSceneView]
     [MovedFrom("UnityEngine.Rendering.LWRP")] public class UniversalAdditionalCameraData : MonoBehaviour, ISerializationCallbackReceiver
     {
-        [Tooltip("If enabled shadows will render for this camera.")]
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs:116]: Why are we removing tooltips?

Comment (martint-unity) [reply com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs:116]: Because the property and Style is dealing with the Tool Tip.

**Comment by martint-unity** [comment]
"What's going on with all the checks for m_AdditionalCameradataSO that are now removed?"
We should only do this once and we are doing it in the beginning of the UI Draw. The same goes for "UpdateProperties()" method, Should only do this at the end of the UI method. It is only needed once afaik.

**Comment by phi-lira** [comment]
@Unity-Technologies/gfx-qa-foundation sorry I merged this without your approval by mistake. 
Please validate and we can address issues in follow up PR.

**Pull request merged**